keras_cv/layers/preprocessing/random_color_degeneration_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+from keras_cv.layers import preprocessing
+
+
+class RandomColorDegenerationTest(tf.test.TestCase):
+    def test_random_color_degeneration_base_case(self):
```

Comment:
we should add more solid test for the actual layer behavior, eg validating the output value.

Reply:
Good point - pretty easy to compute some values using luma anyways.  done.